pages/price-feeds/create-your-first-pyth-app.mdx

```diff
@@ -4,4 +4,7 @@ This tutorial walks through creating a complete application that uses Pyth price
 The application includes both a smart contract in addition to off-chain code that invokes the contract.
 Please choose your environment to get started:
 
-- [EVM](create-your-first-pyth-app/evm)
+**Note** These tutorials are intended for new developers with less smart contract development experience.
```

Comment:
couple minor things here:

* There should be a colon after Note -- it looks weird in the preview otherwise https://documentation-git-pricefeed-tutorial-split-pyth-web.vercel.app/price-feeds/create-your-first-pyth-app
* The wording feels a little too strong. It's written in a very attention-grabbing way right now, and it feels like a serious warning when it's really a gentle nudge. Getting rid of the bold "note" would help. You could also reword it to make it feel like more of a suggestion. The tone should convey "hey doing this tutorial is totally fine, but you may want to check out this other doc"
* It should say "This tutorial" not "these tutorials". I think "Create your first Pyth App" is the tutorial (at least that's how the rest of the doc above is written)